pyproject.toml

```diff
@@ -55,10 +55,10 @@ include = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.8.0"
+python = "^3.9.0"
 jsonschema = "^4.19.1"
 rfc3339-validator = "*" # requred by jsonschema for date-time checker
-jsonschema-specifications = ">=2023.5.2"
+jsonschema-specifications = "^2024.10.1"
```

Comment:
What's the point of `^` when using calendar versioning?  Releases in January of 2025 aren't more likely to include breaking changes than those in December of 2024.

Reply:
You are right, I replaced it with `>=`